### PR TITLE
Initial refactor based on hoodie-store.  Closes Issue #2.

### LINKED ISSUE #2: Update to latest hoodie conventions
See https://github.com/hoodiehq/camp/issues/100

### PR DESCRIPTION
**Pull request by inator**
@gr2m - ready for your initial review! :)

closes #2

### CODE REVIEW

File: example/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+var Hapi = require('hapi')
+var PouchDB = require('pouchdb')
+var memdown = require('memdown')
+var inert = require('inert')
+var path = require('path')
+var EventEmitter = require('events').EventEmitter
+var hapiStore = require('../server')
```
Comment (gr2m) [inline example/index.js]: rename to `hapiTask`

**Review by gr2m — CHANGES_REQUESTED** [review]
I would remove the tests entirely, including all the packages that we only need for testing. We no longer use `@gr2m/frontend-test-setup`, instead we use [selsa](https://www.npmjs.com/package/selsa). I can help setting it up in a follow up PR, you can have a glance at https://github.com/hoodiehq/hoodie-app-tracker/tree/master/test

We also no longer load `"pouchdb"` directly, instead we use [custom builds](https://pouchdb.com/custom.html). So here in our case I think we need `pouchdb-core` and `pouchdb-adapter-memory`, maybe `pouchdb-mapreduce` and `pouchdb-replication`.

Does that all make sense?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,8 @@
 node_modules
-.db
```
Comment (gr2m) [inline .gitignore]: keep the `.db` in here, it is where PouchDB writes its files into. Unless we decide to use `pouchdb-adapter-memory` only

**Comment by gr2m** [comment]
@inator is the review clear? Anything I can help with?

**Comment by inator** [comment]
> @inator is the review clear? Anything I can help with?

@gr2m - Sorry for the delay.  I hope to dive back in this week.

**Comment by inator** [comment]
> We also no longer load "pouchdb" directly, instead we use custom builds. So here in our case I think we need pouchdb-core and pouchdb-adapter-memory, maybe pouchdb-mapreduce and pouchdb-replication.

@gr2m - I'm finally getting back to this.  I based this on [this](https://github.com/hoodiehq/hoodie-store/blob/master/example/index.js#L2).  Do you have an example you can point me to where you do this?

**Comment by gr2m** [comment]
`hoodie-store` is a bit outdated, we don’t use it in our architecture anymore. Here is where we load PouchDB in `hoodie`: https://github.com/hoodiehq/hoodie/blob/master/cli/parse-options.js#L5

You can do sth like this

```js
var PouchDB = require('pouchdb-core')
  .plugin(require('pouchdb-mapreduce'))
  .plugin(require('pouchdb-replication'))
  .plugin(require('pouchdb-adapter-memory'))
```

and add the 4 packages to package.json dependencies, remove `pouchdb`

**Comment by inator** [comment]
@gr2m - I think I addressed all the immediate changes requested.  Hopefully this will get us started with some local testing of tasks. :)

**Review by gr2m — APPROVED** [review]
great start!

**Pull request merged**